### ISSUE TITLE
[BUG] [Java Service]Return "/response/code=400" if parameter validation failure in all APIs.

### ISSUE THREAD

**Issue by lyiyu66** [vmware/singleton#1596]
**Describe the bug**
Return "/response/code=400" if parameter validation failure in all APIs.
Reproducible on all v1/v2 APIs.

**To Reproduce**
1. Go to 'GET
[/i18n/api/v2/formatting/date/localizedDate](http://localhost:8091/i18n/api/doc/swagger-ui.html?urls.primaryName=v2#/operations/formatting-date-api/formatDateUsingGET_2)
2. Set parameter as "pattern=EEEEMMMMdy, longDate=1472728030290, locale=ja/a"
`http://localhost:8091/i18n/api/v2/formatting/date/localizedDate?locale=ja.%2Faf&longDate=1472728030290&pattern=EEEEMMMMdy`
4. See error:
```
{
  "response": {
    "code": 400,
    "message": "Incorrect locale(only allows letter, number, dot, underline, dashline)",
    "serverTime": ""
  },
  "signature": "",
  "data": null
}
```

**Expected behavior**
Should get code=400 as before.
```
{
"code" : 400,
"message" : "Incorrect locale(only allows letter, number, dot, underline, dashline)",
"serverTime" : ""
}'
```

**Comment by tigershi** [comment]
this is fixed the issue https://github.com/vmware/singleton/issues/1597

**Comment by lyiyu66** [comment]
The issue is fixed with commit c884e917476a49402f68de24cf67ce3d0e9a2d64
